integration/src/test/scala/org/knora/webapi/core/LayersTest.scala

```diff
@@ -114,7 +115,8 @@ object LayersTest {
       ConstructTransformer.layer,
       DspIngestClientLive.layer,
       GravsearchTypeInspectionRunner.layer,
-      GroupsResponderADMLive.layer,
+      GroupsResponderADM.layer,
+      GroupsRestService.layer,
```

Comment:
```suggestion
```
**issue**: Creating the same service twice

This service is already present in the [AdminApiModule](https://github.com/dasch-swiss/dsp-api/blob/main/webapi/src/main/scala/org/knora/webapi/slice/admin/api/AdminApiModule.scala#L54)